internal/graph/graph.go

```diff
@@ -34,6 +34,8 @@ var (
 	// Removes package name and method arugments for Go function names.
 	// See tests for examples.
 	goRegExp = regexp.MustCompile(`^(?:[\w\-\.]+\/)+(.+)`)
+	// Checks for a package name that could be a module version.
+	goVerRegExp = regexp.MustCompile(`^v[2-9]+\.`)
```

Comment:
Oh, you're right, I brainfarted on that one. Should have been `^v([2-9]|[0-9]+)\.`.

Reply:
I switched it to a corrected regex, but if there's some more complicated one that works better it can be used. I know my brain shuts down trying to read that long one... :slightly_smiling_face:

Reply:
The key thing is to allow v2, v3, v10, v1234, etc, but not v0 or v1.

Reply:
```
(?:^(?:[\w\-\.]+\/)+((?:[\w\-\.]+\/v(?:[2-9]|[1-9][0-9]+)+)(?:\.[^.\n]+){2})$)|(?:^(?:[\w\-\.]+\/)+(.+))
```

Is the long one with the restricted version select; that'd be a one line change and it appears to pass the tests. I'm happy to use it instead.